docs/TESTS.md

```diff
@@ -1,15 +1,20 @@
 # Tests
 
-<!-- TODO: write document
+To run the tests for your SQLite exercises, follow these steps:
 
-  This document should describe everything related to running tests in the track.
+1. Open your preferred terminal.
+2. Navigate to the directory containing the appropriate `STUB_test.sql` file, where `STUB` is the name of the exercise, using hyphens instead of spaces and all lowercase.
```

Comment:
Rather than `STUB` does it make sense to use variables? `${stub}`

Reply:
I don't know if that is less or more confusing for people who don't know that syntax?

Reply:
I think `${stub}` is a pretty clear placeholder, even if the student isn't familiar with shell syntax.